app/resources/scanned_resources/letter_change_set.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class LetterChangeSet < ChangeSet
+  apply_workflow(DraftCompleteWorkflow)
+  core_resource(change_set: "letter")
+  enable_order_manager
+  enable_pdf_support
+
+  property :sender, multiple: false, required: false, form: NameWithPlaceChangeSet, populator: :populate_name_with_place
+  property :recipient, multiple: false, required: false, form: NameWithPlaceChangeSet, populator: :populate_name_with_place
+
+  def primary_terms
+    feature_terms.dup.insert(
+      2,
+      [
+        :sender,
+        :recipient,
+        :member_of_collection_ids
+      ]
+    ).flatten
+  end
+
+  def prepopulate!(_args = {})
+    self.sender ||= NameWithPlace.new
+    self.recipient ||= NameWithPlace.new
+    super
+  end
+
+  def populate_name_with_place(fragment:, as:, **)
```

Comment:
Why does this live here instead of the name with place change set? Is the idea that different change sets may want to prepopulate it differently?

Reply:
I see fragment in the reform documentation; where is `as` coming from?

Reply:
This is just how the reform documentation says to populate a nested object - I don't think there's a way to do it on the ChangeSet, especially to ignore population if the params are empty.

`as` is just one of the arguments, I noticed it when I was fiddling. I'm having a hard time finding it documented somewhere, but it's definitely passed in.